src/network/wire/lib.zig

```diff
@@ -88,14 +88,16 @@ pub fn receiveMessage(allocator: std.mem.Allocator, r: anytype) !protocol.messag
     const checksum = try r.readBytesNoEof(4);
 
     // Read payload
-    const message: protocol.messages.Message = if (std.mem.eql(u8, &command, protocol.messages.VersionMessage.name()))
+    var message: protocol.messages.Message = if (std.mem.eql(u8, &command, protocol.messages.VersionMessage.name()))
```

Comment:
Whi is this changing?

Reply:
Because in deinit we're not expecting a const, and zig complains
It will be removed by the slice instead of ArrayList